Sources/EpoxyCollectionView/CollectionView/Internal/ReuseIDStore.swift

```diff
@@ -25,18 +25,46 @@ public final class ReuseIDStore {
 
   /// Vends a new reuse identifier string for the given `ViewDifferentiator`, generating a new reuse
   /// identifier whenever a new `viewDifferentiator` is encountered as determined by its equality.
-  public func reuseID(for viewDifferentiator: ViewDifferentiator) -> String {
+  public func registerReuseID(for viewDifferentiator: ViewDifferentiator) -> String {
```

Comment:
and maybe this can be `public func reuseID(byRegistering viewDifferentiator: ViewDifferentiator) -> String`